feat: Groups: Add "Followers" page for Owners/Organizers

MeghalBisht opened the pull request:
<!-- 
(Thanks for sending a pull request! Please make sure you click the link above to view the contribution guidelines, then fill out the blanks below.)
-->

<!-- Add the issue number that is fixed by this PR (In the form Fixes #123) -->

Fixes parts of #7601 
To be followed up -  CSV export part

#### Short description of what this resolves:
![Screenshot from 2021-07-29 03-38-28](https://user-images.githubusercontent.com/61330148/127402593-9dcaefdd-d101-4cb8-b807-ca34d66fc151.png)
![Screenshot from 2021-07-29 03-38-24](https://user-images.githubusercontent.com/61330148/127402600-9c66d5dc-f331-4832-9136-880cb1c07f9e.png)



#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes <!-- use `ember test` to run all the tests -->
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

maze-runnar commented:
![Screenshot from 2021-07-27 22-37-06](https://user-images.githubusercontent.com/56407566/127197560-d380858e-4ee4-47dd-88ea-30713bbd714e.png)

maze-runnar commented:
@MeghalBisht you will create new PR for CSV export, right?

MeghalBisht commented on app/templates/groups/edit/followers.hbs:
```diff
@@ -0,0 +1,16 @@
+<GroupNav @group={{this.model.group}} />
```
@maze-runnar After adding the navigation, if you try to save the event settings or events to the group it throws an error.

maze-runnar replied:
Oh, I will check and try to fix it. If not possible then will revert.

maze-runnar commented:
After visiting to followers route, whether the group-nav is added or not , saving event is giving error. And i think followers should not be in edit section.

mariobehling commented:
I think the slugs we are implementing do now always make sense, but I do not have the perfect solution right now. In any case the followers list will be editable in some ways by organizers, but these are future issues (e.g. organizers should be able to throw out a user). There could also be a public followers list, but again it is a future issue. So, this looks like a good approach to me in general.

mariobehling commented:
I missed a point in the issue description. We need a "followers icon" behind the "Events" icon on the groups overview page. The followers icon should be different to the team icon. Could you add it, please as well?

![Screenshot from 2021-07-28 13-44-11](https://user-images.githubusercontent.com/1583873/127316927-a1cbb003-b332-4d6b-b3d8-bf711e06d585.png)

I see that the page is not working on the test deployment. I guess it is because we are using the production API server for testing. Have you been able to see how the PR frontend deployments on Vercel could be changed to use https://test-api.eventyay.com instead?

maze-runnar commented:
@MeghalBisht I have changed the path for the followers page and added follower icon. if it's ok, I can push changes?

![Screenshot from 2021-07-28 22-25-09](https://user-images.githubusercontent.com/56407566/127364505-e617902d-5fb7-4b63-a620-375b29635991.png)
![Screenshot from 2021-07-28 22-25-05](https://user-images.githubusercontent.com/56407566/127364513-33afd3fc-be05-4555-910a-da3511120886.png)

MeghalBisht commented:
> @MeghalBisht I have changed the path for the followers page and added follower icon. if it's ok, I can push changes?

I tried this too. But after you visit follower's page and again try to go back to edit events or settings page, the patch request fails. Did you check that?

maze-runnar commented:
> > @MeghalBisht I have changed the path for the followers page and added follower icon. if it's ok, I can push changes?
> 
> I tried this too. But after you visit follower's page and again try to go back to edit events or settings page, the patch request fails. Did you check that?
> 
> 

Yeah, error is not solved 😅
